src/kanata/windows/llhook.rs

```diff
@@ -65,9 +73,14 @@ impl Kanata {
             try_send_panic(&preprocess_tx, key_event);
             true
         });
+        #[cfg(all(target_os = "windows", feature = "gui"))]
+        let _ui = build_tray(&_kanata)?;
 
-        // The event loop is also required for the low-level keyboard hook to work.
-        native_windows_gui::dispatch_thread_events();
+        native_windows_gui::dispatch_thread_events(); // The event loop is also required for the low-level keyboard hook to work.
+                                                      // if *IS_TERM  {
+                                                      // eprintln!("\nPress enter to exit"); // moved from main to not panic on a disconnected channel
```

Comment:
I didn't understand the need for this eprintln in the main code, so I left it commented out instead of removing

This had the benefit in the main script of not echoing back pressed keys, but than this was due to another feature `Init: catching only releases and sending immediately` which I also didn't understand the purpose of :))